test/e2e/autoscale_test.go

```diff
@@ -345,6 +411,33 @@ func TestAutoscaleUpCountPods(t *testing.T) {
 	}
 }
 
+func TestRPSBasedAutoscaleCountPods(t *testing.T) {
+	t.Parallel()
+	cancel := logstream.Start(t)
+	defer cancel()
+
+	ctx := setup(t, autoscaling.KPA, autoscaling.RPS,
+		rtesting.WithConfigAnnotations(map[string]string{
+			autoscaling.TargetAnnotationKey:            "10",
+			autoscaling.TargetUtilizationPercentageKey: "70",
+		}))
+	ctx.targetUtilization = 0.7
```

Comment:
please infer from the ConfigMap.

Reply:
I guess there's no need to as its overridden in the annotations.

Reply:
There's `ontainer-concurrency-target-percentage` in configmap, but no flag for `targetUtilization` of RPS. We only support default value or setting it in annotation. Since we're using annotations here, I'd prefer to explicitly set the value to match the annotations.

Reply:
Removed as making target and tu as required args.